web/src/pages/Cases/CaseDetails/Appeal/Classic/StageExplainer.tsx

```diff
@@ -31,24 +35,63 @@ const CountdownLabel = styled.label`
 `;
 
 interface IStageExplainer {
-  loserSideCountdown: number | undefined;
+  countdown: number | undefined;
+  stage: 1 | 2;
 }
 
-const StageExplainer: React.FC<IStageExplainer> = ({ loserSideCountdown }) => (
-  <StyledBox>
-    <CountdownLabel>
-      <HourglassIcon />
-      {typeof loserSideCountdown !== "undefined" && secondsToDayHourMinute(loserSideCountdown)}
-    </CountdownLabel>
+const StageOneExplanation: React.FC = () => (
+  <div>
+    {" "}
+    <label>
+      Losing options can only be funded <small>before</small> the deadline.
+    </label>
+    <label>
+      If no losing option is <small>fully funded</small> in time, the jury decision is maintained.
+    </label>
+  </div>
+);
+
+const StageTwoExplanation: React.FC = () => {
+  const { fundedChoices } = useFundingContext();
+  const options = useOptionsContext();
+  return (
     <div>
       <label>
-        Losing options can only be funded <small>before</small> the deadline.
+        Loser deadline has <small>finalized</small>, you can only fund the current winner.
+      </label>
+      <label>
+        The sum of funds must reach 100%. If it's not fully funded in time the option fully funded at{" "}
+        <small>stage 1</small> is declared the winner of the case.{" "}
```

Comment:
This is slightly inaccurate, for the current implementation of `DisputeKitClassic` the sum of funds is not considered, the only condition to make an appeal go through is to completely fund two options. So I would write something like this:
```If the current winner is not fully funded in time, the option funded during the previous stage will be declared as the final winner.```